{
   "assignee" : null,
   "body" : "I get this error when I try to send BTC\r\nA FATAL ERROR OCCURRED. Bitcoin can no longer continue safely and will quit.\r\nException: St13runtime_error\r\nCDB(): canÃ¢ÂÂt open database file wallet.dat, error -30974\r\nC:\\program files (x86)\\Bitcoin\\bitcoin-qt.exe in Runaway exception\r\n\r\n\r\nAny help would be greatly appreciated\r\nThanks",
   "closed_at" : "2014-04-05T02:46:35Z",
   "closed_by" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/6610971?v=3",
      "events_url" : "https://api.github.com/users/steelmythos/events{/privacy}",
      "followers_url" : "https://api.github.com/users/steelmythos/followers",
      "following_url" : "https://api.github.com/users/steelmythos/following{/other_user}",
      "gists_url" : "https://api.github.com/users/steelmythos/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/steelmythos",
      "id" : 6610971,
      "login" : "steelmythos",
      "organizations_url" : "https://api.github.com/users/steelmythos/orgs",
      "received_events_url" : "https://api.github.com/users/steelmythos/received_events",
      "repos_url" : "https://api.github.com/users/steelmythos/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/steelmythos/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/steelmythos/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/steelmythos"
   },
   "comments" : 1,
   "comments_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/4002/comments",
   "created_at" : "2014-04-04T23:04:45Z",
   "events_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/4002/events",
   "html_url" : "https://github.com/bitcoin/bitcoin/issues/4002",
   "id" : 30901498,
   "labels" : [],
   "labels_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/4002/labels{/name}",
   "locked" : false,
   "milestone" : null,
   "number" : 4002,
   "repository_url" : "https://api.github.com/repos/bitcoin/bitcoin",
   "state" : "closed",
   "title" : "NewBie -- Cannot send from wallet  HELP please",
   "updated_at" : "2014-04-05T02:46:35Z",
   "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/4002",
   "user" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/6610971?v=3",
      "events_url" : "https://api.github.com/users/steelmythos/events{/privacy}",
      "followers_url" : "https://api.github.com/users/steelmythos/followers",
      "following_url" : "https://api.github.com/users/steelmythos/following{/other_user}",
      "gists_url" : "https://api.github.com/users/steelmythos/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/steelmythos",
      "id" : 6610971,
      "login" : "steelmythos",
      "organizations_url" : "https://api.github.com/users/steelmythos/orgs",
      "received_events_url" : "https://api.github.com/users/steelmythos/received_events",
      "repos_url" : "https://api.github.com/users/steelmythos/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/steelmythos/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/steelmythos/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/steelmythos"
   }
}
